mautic/mautic
# Remove email config from installer

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [❌]
| New feature/enhancement? (use the a.x branch)      | [✅]
| Deprecations?                          | [❌]
| BC breaks? (use the c.x branch)        | [✅]
| Automated tests included?              | [✅] 
| Related user documentation PR URL      |  
| Related developer documentation PR URL |  
| Issue(s) addressed                     |  



#### Description:


This PR removes the email configuration step from both web and CLI installers which will ease implementing the [new mailing service](https://github.com/mautic/mautic/issues/12096) utilizing Symfony mailer.

#### Steps to test this PR:


1. Check if the web installer no longer contains the email step and is working as expected.
2. Check if the CLI installer no longer contains the email step / options and is working as expected. 



#### BC breaks:

1. The web installer no longer contains the email step.
2. The CLI installer no longer contains the email step and the email options.
